release_scripts/signassert.py

```diff
@@ -0,0 +1,63 @@
+################################################################################
+#                                                                              #
+# Copyright (C) 2011-2015, Armory Technologies, Inc.                           #
+# Distributed under the GNU Affero General Public License (AGPL v3)            #
+# See LICENSE or http://www.gnu.org/licenses/agpl.html                         #
+#                                                                              #
+################################################################################
+
+import getpass
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
+
+from armoryengine.ALL import *
```

Comment:
I tried changing that line to just be `from armoryengine.PyBtcWallet import *`. The problem is that a lot of the code (including PyBtcWallet) imports everything from ArmoryUtils. So even if I am careful to just import exactly what I need, everything from ArmoryUtils will still be imported indirectly by other Armory code.

So while I see why the argument parsing is part of ArmoryUtils (because ArmoryQt and armoryd shares the same arguments), that kind of structure seems to make it impossible to have a 3rd program/script that uses Armory code, but has its own unique arguments.